src/compute/encryption.rs

```diff
@@ -500,7 +505,7 @@ FQIDAQAB
             fs::read(&aes_key_file_in_dir).expect("Failed to read AES key file from dir");
         assert!(!aes_key_content_bytes.is_empty());
         // AES key is now stored as raw encrypted bytes, not Base64 string
-        assert_eq!(aes_key_content_bytes.len(), 256); // RSA-2048 produces 256-byte output
+        assert_eq!(aes_key_content_bytes.len(), 256); // RSA-2048 produces 256-bits output
```

Comment:
can we check this ?
Here, `aes_key_content_bytes` seems to refer to the content of the file holding the **RSA encrypted AES key**
The content may be 256 bytes, i.e 2048 bits